src/test/osd/TestMClockScheduler.cc

```diff
@@ -46,8 +54,9 @@ class mClockSchedulerTest : public testing::Test {
     is_rotational(false),
     cutoff_priority(12),
     monc(nullptr),
+    init_perfcounter(false),
```

Comment:
After remove the logger object in ~mClockScheduler you can make it true for test case.